Hackaton2023IOS/Data/Models/Category.swift

```diff
@@ -0,0 +1,44 @@
+//
+//  Category.swift
+//  Hackaton2023IOS
+//
+//  Created by Andrii Lytvyniuk on 14.05.2023.
+//
+
+import Foundation
+
+struct Category: Decodable {
+    struct SubCategory: Decodable {
+        let subCategory: String
+        let current: String
+        let required: String
+    }
+    let category: String
+    let subCategories: [SubCategory]
+}
+
+extension Category.SubCategory {
+    init(subCategory: String, current: Level, required: Level) {
+        self.subCategory = subCategory
+        self.current = current.rawValue
+        self.required = required.rawValue
+    }
+}
+
+enum Level: String, RawRepresentable {
```

Comment:
+1 for RawRepresentable

Reply:
Anyway for string we don't need it to specify directly https://swiftdoc.org/v4.2/protocol/rawrepresentable/